website/docs/cli/commands/docs/docs-generate.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: atmos docs generate
+sidebar_label: generate
+sidebar_class_name: command
+description: "Generate readme subcommand for Atmos docs"
+id: generate
+---
+import Screengrab from '@site/src/components/Screengrab'
+
+:::note Purpose
+
+Use this command to generate a README.md from one or more input sources.  
+
+:::
+
+The sources are defined at `docs.generate.readme` section of `atmos.yaml`.
+
+    ```yaml
+    docs:
+    generate:
+      # Generate README in current working directory
+      readme:
+        base-dir: .
+        input:
+          - "./README.yaml"
+        # A template can be a remote URL/github/gitlab/etc 
+        template:  "https://raw.githubusercontent.com/cloudposse/.github/5a599e3b929f871f333cb9681a721d26b237d8de/README.md.gotmpl"
+        # The final README
+        output: "README.md"
+        terraform:
+          source: src/
+          enabled: false
+          format: "markdown"
+          show_providers: false
+          show_inputs: true
+          show_outputs: true
+          sort_by: "name"
+          hide_empty: false
+          indent_level: 2
+
+    ```
+
+The command combines all local or remote YAML files specified at `input` and template file then generates README.md at the `output` folder. In case the template contains `terraform_docs` key, e.g.  
+
+    ```yaml
+{{- $data := (ds "config") -}}
+
+{{ $data.name | default "Project Title" }}
+
+{{ $data.description | default "No description provided." }}
+
+{{ if has $data "extra_info" }}
+Extra info: {{ $data.extra_info }}
+{{ end }}
+
+{{ if has $data "terraform_docs" }}
+## Terraform Docs
+{{ $data.terraform_docs }}
+{{ end }}
+
+    ```
+the resultant README.md will also have a corresponding section rendered. By default `terraform.format` is set to `markdown table`, and can also be `markdown`, `tfvars hcl`, and `tfvars json`.  
+This command also aliases to “atmos generate docs”.
```

Comment:
the alias, `atmos generate docs` doesnt seem to work:

```bash
$ atmos generate docs
 Incorrect Usage

The command  atmos generate generate  requires a subcommand
Valid subcommands are:
• readme
Run  atmos generate generate --help  for usage
```

Reply:
@milldr the subcommand is configured as `readme`

```
The command  atmos generate generate  requires a subcommand
Valid subcommands are:
• readme
```

Reply:
The `readme` name is arbitrary (a configuration), and multiple configurations can exist.

Reply:
yes that works, but the docs say that we have an alias too as `atmos generate docs`. We should either fix or remove that command

Reply:
oh I see. it's either `atmos docs generate` or `atmos generate docs`. But both cases need a subcommand, `readme`

Reply:
that works, but to me it's redundant to have both `docs` and `generate`. Plus it's confusing with the other `atmos docs` functionality

Reply:
Yes, so the idea was that we would bundle this as part of `atmos generate` functionality, but my oversight is the existing generate functionality is part of `atmos terraform generate`, which is unrelated. 

So since we don't have that yet, lets consolidate this under `atmos docs generate`. That said, we still require the subcommand, because different configurations are required to generate docs for components, vs generating docs for say the root of the project. That's why it's configurable. 

In the future, we may move this underneath `atmos generate` and alias `atmos docs generate` to `atmos generate`.